Gemfile.lock

```diff
@@ -36,7 +36,7 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.1.4.1)
+    activesupport (6.1.6.1)
```

Comment:
We cannot change this with `activesupport (7.0.3.1)` easily as we need to test with Ruby 2.5 and 2.6 but activesupport 7.0 supports Ruby 2.7.0 or higher.